### DISCUSSION TITLE
Can't remove DRM with version 6.8.1

### DISCUSSION THREAD

**Discussion by ILoveSkilledCoders** [apprenticeharper/DeDRM_tools#1919] [Q&A]
Hi,

I'm on Win 7 & so I can't have the latest version of Calibre.

The plugin 6.8.1 is installed on Calibre, but it won't remove it unless there's a step before converting it to a different file that I'm missing.

I don't understand that mile long page of troubleshooting & I don't want to do command prompt by myself.

It's an Amazon book that I paid for.

Please help, I really need to print off several of the pages of the book as there are instructions.

Thanks

**Comment by ElleKayEm** [comment]
These are your 3 options:
* Find an earlier version of Kindle for PC.  No later then version 1.26.
* Turn off KFX downloads from Kindle for PC.
* If you have an an E-ink Kindle (not a Fire), you can use the "download & transfer via USB" option from the website.

**Reply by ILoveSkilledCoders** [reply]
Hi there,

I don't even know what Kindle for PC is. 

Yes, I have a very old Kindle 3 and I added the book from the documents folder from the Kindle to Calibre using a USB cord. Is that what you mean?

That's what I'm supposed to do, right?

Thanks for trying to help.

**Reply by ElleKayEm** [reply]
The Kindle Keyboard? Yes, you can get the book directly from the Kindle in that case. But you will need to put its serial number into DeDRM's customization.

**Reply by ILoveSkilledCoders** [reply]
LOL, yes that one.

Where do I put the serial number? 

I added the plugin to Calibre & then added the book to Calibre, then tried to convert.

Nothing asked me for any serial number. You mean the serial number of my Kindle or the book?

Thanks again.

**Reply by Leseratte10** [reply]
Serial number of the Kindle. Go into the Calibre settings, click "Plugins", click on the DeDRM plugin, click "Customize Plugin", then click "eInk Kindle ebooks", then click the green "plus" icon and enter your Kindle's serial number. Then delete the book from Calibre and import it again.

**Reply by ILoveSkilledCoders** [reply]
@Leseratte10 & @ElleKayEm - it worked.

Thank you VERY VERY MUCH!!!
